BUILD

```diff
@@ -31,12 +31,15 @@ cc_library(
         "src/tim/vx/tensor_private.h",
         "src/tim/vx/type_utils.h",
         "src/tim/vx/type_utils.cc",
+        "src/tim/vx/test_utils.h",
```

Comment:
These two files should be move to unit_test module, because "tim-vx_interface" doesn't need them.

Reply:
Since we don't put all test file in to unit test module. We have two option for this:
1. Keep test util close to where it used.
2. Move all unit_test to standalone module

Currently, I'd prefer 1st option since we don't want to move unit test to standalone moudle because we also need add unit test for private apis not only public ones.